### PR TITLE
doc (executorExec): multiline

### PR DESCRIPTION
**Pull request by yesnault**
And lowercase for att json / yaml to avoid confusion

### CODE REVIEW

File: executors/exec/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+err
+code
+timeseconds
+timehuman
+```
+
+- result.timeseconds & result.timehuman: time of execution
+- result.executor.path: executor condition
+- result.err: if exists, this field contains error
+- result.body: body of HTTP response
```
Comment (fsamin) [inline executors/exec/README.md]: http ?

File: executors/exec/README.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+stdout
+stderr
+err
+code
+timeseconds
+timehuman
+```
+
+- result.timeseconds & result.timehuman: time of execution
+- result.executor.path: executor condition
```
Comment (fsamin) [inline executors/exec/README.md]: not sure to understand

File: executors/exec/README.md (hunk 1 above)
Comment (yesnault) [inline executors/exec/README.md]: fixed

File: executors/exec/README.md (hunk 2 above)
Comment (yesnault) [inline executors/exec/README.md]: fixed

**Pull request merged**